migrations/20200627143142-send-fiscal-host-updates-notification.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface
+      .addColumn('Updates', 'updateNotificationAudience', {
```

Comment:
We don't need to repeat the word "update" here, because we're in the "Update" table (like you wouldn't do `Comment.commentTags`, just `Comment.tags`).

```suggestion
      .addColumn('Updates', 'notificationAudience', {
```